js/main.js

```diff
@@ -222,21 +222,22 @@ var mapView = {
 
   },
   initMap: function() {
-    var self = this;
+    var self = this,
+      mapStyle1 = [ 
```

Comment:
i think the constant definitions at the top is better,
and the name `PokemonGoMapStyle` is better than `mapStyle1`